utils/resolve.js

```diff
@@ -235,4 +235,6 @@ function resolve(p, context) {
   }
 }
 resolve.relative = relative;
+resolve.resolverReducer = resolverReducer;
+resolve.requireResolver = requireResolver;
```

Comment:
it'd be ideal to add some unit tests for these. i know there aren't any direct tests of the utils package, but it might be time to add some.

Reply:
I've added some now. What do you think?